### PR TITLE
Bulk assign

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Add bulk assign page (/assign-calls)

image

It allows to assign unassigned calls and reassign assigned calls in which case it favours the person that the calls are already assigned to, provided that person is one of the selected call handlers.

### CODE REVIEW

File: src/pages/assign-calls.js [hunk 1]
```diff
@@ -1,69 +1,245 @@
...
-               
-                </div>
+    const router = useRouter();
+    const [callHandlers, setCallHandlers] = useState([]);
+    const [selectedCallHandlers, setSelectedCallHandlers] = useState([]);
+    const [selectedCallType, setSelectedCallType] = useState('All');
+    const [selectedAssignment, setSelectedAssignment] = useState([]);
+    const [errorsExist, setErrorsExist] = useState(null);
+
+    const callTypes = ['All', 'Help Request', 'CEV', 'Welfare Call', 'Contact Tracing'];
```
Comment (maysakanoni) [inline src/pages/assign-calls.js:19]: maybe we can extract this into the constants file, i think its being used in multiple places

**Review by maysakanoni — COMMENTED** [review]
This works really nicely! As I was testing it out i bulk assigned- but then wanted to bulk un-assign which I think is out of scope but i was wondering if the call handlers would ever want to do that. So say they bulk assigned to All but A was away, how do they unassign cases assigned to person A- or would this have to be done though single assign

**Comment by kosiakkatrina** [comment]
@maysakanoni they could reassign the cases to someone else, but I see how un-assigning it might make more sense

File: src/helpers/constants.js [hunk 2]
```diff
@@ -9,3 +9,5 @@ export const callOutcomes = {
   voicemail: "Voicemail left" ,
   wrong_number: "Wrong number",
   no_answer_machine: "No answer machine" }
+
+export const callTypes = ['All', 'Help Request', 'CEV', 'Welfare Call', 'Contact Tracing'];
```
Comment (maysakanoni) [inline src/helpers/constants.js:13]: 👀

**Review by maysakanoni — APPROVED** [review]
Nice work- i haven't checked in in my branch again but saw it worked earlier(i have a 121 now so dont want to hold u up)

**Pull request merged**